File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_6.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
a = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
b = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
c = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
d = gets.to_i
if (a <= 0) || (b <= 0) || (c <= 0) || (d <= 0)
    puts ('Necorrectnie dannie')
elsif (a == c) & (b == d)
    puts ('Mojno postroit pryamoygolnic s takimi storonami')
else
    puts ('Nelzya postroit pryamoygolnic s takimi storonami')
end

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_6.go
package main
import ("fmt"
        
)

func main() {
    var a float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&a)
    var b float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&b)
    var c float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&c)
    var d float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&d)
    if (a <= 0) || (b <= 0) || (c <= 0) || (d <= 0) {
        fmt.Print ("Nekorrektnie dannie")
    } else if (a == c) && (b == d) {
        fmt.Print ("Mojno postroit pryamoygolnik s takimi storonami")
    } else {
        fmt.Print ("Nelzya postroit pryamoygolnik s takimi storonami")
    }
}